config/application.rb

```diff
@@ -28,13 +28,22 @@
 # default values for some ENV variables
 ENV['APPLICATION'] ||= "mds"
 ENV['HOSTNAME'] ||= "mds.local"
-ENV['API_URL'] ||= "https://api.test.datacite.org"
-ENV['REALM'] ||= "mds.test.datacite.org"
+ENV['API_URL'] ||= "https://api.stage.datacite.org"
+ENV['REALM'] ||= "mds.stage.datacite.org"
 ENV['MEMCACHE_SERVERS'] ||= "memcached:11211"
 ENV['SITE_TITLE'] ||= "MDS API"
 ENV['LOG_LEVEL'] ||= "info"
 ENV['TRUSTED_IP'] ||= "10.0.90.1"
 
+def mds_url
```

Comment:
This is a local variable. Maybe define as ENV variable?

Reply:
Hmm, well it's more a global function, as that's why I moved it into the config so it was available to the the entire application, or at least that's how I understand rails config.
Admittedly there isn't a huge need for it to be like this, I was just keeping it to what it was but making it more available.

I'll change it to be an ENV variable though to be consistent with the other ENV stuff happening here.